geth/txqueue/txqueue_manager_test.go

```diff
@@ -196,16 +237,17 @@ func (s *TxQueueTestSuite) TestAccountMismatch() {
 }
 
 func (s *TxQueueTestSuite) TestInvalidPassword() {
-	s.accountManagerMock.EXPECT().SelectedAccount().Return(&common.SelectedExtKey{
-		Address: common.FromAddress(TestConfig.Account1.Address),
-	}, nil)
-
-	s.nodeManagerMock.EXPECT().NodeConfig().Return(
-		params.NewNodeConfig("/tmp", params.RopstenNetworkID, true),
-	)
+	password := "invalid-password"
+	key, _ := crypto.GenerateKey()
+	account := &common.SelectedExtKey{
+		Address:    common.FromAddress(TestConfig.Account1.Address),
+		AccountKey: &keystore.Key{PrivateKey: key},
+	}
+	s.setupStatusBackend(account, password)
 
-	// Set ErrDecrypt error response as expected with a wrong password.
-	s.nodeManagerMock.EXPECT().LightEthereumService().Return(nil, keystore.ErrDecrypt)
+	nonce := hexutil.Uint64(10)
+	gas := hexutil.Big(*big.NewInt(defaultGas + 1))
+	s.setupTransactionPoolAPI(account, nonce, gas, keystore.ErrDecrypt)
```

Comment:
This test does not test a real scenario. `keystore.ErrDecrypt` is returned by `SendRawTransaction` while it should be by `VerifyAccountPassword`. I would recommend to change that and also make sure that `SendRawTransaction` is not called.